akka-http-core/src/main/scala/akka/http/impl/engine/ws/WebSocket.scala

```diff
@@ -7,6 +7,7 @@ package akka.http.impl.engine.ws
 import java.util.Random
 import akka.NotUsed
 import akka.event.LoggingAdapter
+import akka.stream.impl.fusing.GraphStages.SimpleLinearGraphStage
```

Comment:
I think I'd personally rather have a bit of repeated code using the public API than introduce a dependency on stream internal here.